src/tools/fuzzing.h

```diff
@@ -428,7 +428,8 @@ class TranslateToFuzzReader {
       Type type = getConcreteType();
       std::vector<Type> params;
       params.push_back(type);
-      Index numValues = upToSquared(MAX_PARAMS - 1);
+      Index numValues =
+        wasm.features.hasMultivalue() ? upTo(1) : upToSquared(MAX_PARAMS - 1);
```

Comment:
`upTo(1)` is always 0, so can just put that.

Btw, how about moving the `+ 1` from the next line to here (so it's `? 1 : (upToSquared(MAX_PARAMS - 1) + 1);`? That way the name `numValues` is more accurate (I was confused by this for a bit earlier today).

Reply:
Oh, I meant `upTo(2)` then.

Reply:
Oh wait no I didn't. I also got confused by the +1 😂

Reply:
And also the conditional values are in the wrong order. What a mess.